GL.cs

```diff
@@ -101,6 +101,8 @@ public unsafe static class GL
     /// </summary>
     public static readonly void* NULL = (void*)0;
 
+    static GLDEBUGPROC callbackUnsafe;

```

Comment:
Consider making this `private` and calling it something like `_unsafeDebugCallbackToPreventGC` or similar :) Stuff like this often gets confused further down the line as time passes hehe, so I like longer names that explain them + bonus is that no `// comment that describes it` is needed :) Also, consider moving the declaration of this field down to where `GLDEBUGPROC` is declared, so everything is close together :)

Good catch though - I encountered these things as well back when I was working on [DotGLFW](https://github.com/dcronqvist/DotGLFW), and my solution was identical 👍 thank you for your contribution, friend!

Reply:
yeah, i did what you told me to do. i pushed changes to my fork. i dont know if there is anything else i need to do more. this is my first pull request so please tell me if i need to do something more(idk maybe i missed something). thank you too. i absolutely love what you do, i started learning opengl using dotgl and dotglfw, i appreciate it.